user-docs/modules/ROOT/pages/bootc-setup.adoc

```diff
@@ -0,0 +1,72 @@
+= Getting Started with IoT Bootable Containers (bootc)
+
+Since the launch of Fedora 41, Fedora IoT has supported bootable containers (bootc).
+Bootable containers are similar in function to other containers (like Podman), but allow the user to boot into the container as an immutable system.
+Any updates or package installs are then layered on top of the base bootc image, allowing rollbacks in the event of an issue.
+Each bootc image ships with a kernel, bootloader, drivers, etc, rendering them fully bootable and not requiring a container manager (Podman, Kubernetes, etc).
```

Comment:
This is a bit weird, you would/could still use a container manager such as podman to maintain applications, the container in the bootc case is mechanism for maintaining the base OS.